client/test/data/membershipRequests.js

```diff
@@ -0,0 +1,48 @@
+import faker from 'faker';
+
+// Example Membership Request data used for testing with no randomization
+export const mockedMembershipRequests = [
+  {
+    id: 1,
+    name: 'John Doe',
+    requestedDate: '2022-12-05',
+    note: 'Please process this request as soon as possible'
+  },
+  {
+    id: 2,
+    name: 'Jane Smith',
+    requestedDate: '2022-11-27',
+    note: 'This request can be postponed for now.'
+  },
+  {
+    id: 3,
+    name: 'William Brown',
+    requestedDate: '2022-12-01',
+  },
+  {
+    id: 4,
+    name: 'Emma Wilson',
+    requestedDate: '2022-11-20',
+  },
+  {
+    id: 5,
+    name: 'Micheal Johnson',
+    requestedDate: '2022-11-30'
+  }
+];
+
+// Randomized membership request data
+export const createMockedMembershipRequests = (number) => {
+  const mockedRequests = [];
+
+  for (let i = 0; i <= number; i++) {
+    mockedRequests.push({
+      id: i + 5,
+      name: faker.name.findName(),
```

Comment:
```suggestion
      userNameWithCssId: faker.name.findName(),
```
I think the attribute names needs to be changed to `userNameWithCssId` for the names to appear on Storybook.